internal/compiler/passes/const_propagation.rs

```diff
@@ -207,6 +207,7 @@ fn simplify_expression(expr: &mut Expression) -> bool {
         Expression::LayoutCacheAccess { .. } => false,
         Expression::SolveLayout { .. } => false,
         Expression::ComputeLayoutInfo { .. } => false,
+        Expression::DebugHook { .. } => false, // This is not const by design
```

Comment:
The return value is actually whether this contains a referenced element, not if it is const.

The inner expression can still be simplified, so the fall trough seems correct.

Or why did you add this?

Reply:
<That's gone now. I misunderstood what this does.